### PR TITLE
Migrate RabbitMQ from bitnami

### PR DESCRIPTION
**Pull request by roee-shapira-tensorleap**
- Remove rabbitmq bitnami chart
- Add custom rabbitmq chart templates
- Remove old rabbitmq headless refs
- Change to use default rabbitmq user

### CODE REVIEW

**Comment by roee-shapira-tensorleap** [comment]
Screen Shot 2022-09-02 at 13 11 40

**Review by yotamazriel — APPROVED** [review]
LGTM

File: charts/tensorleap/templates/rabbitmq-sts.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+              valueFrom:
+                configMapKeyRef:
+                  name: rabbitmq-config
+                  key: erlangCookie
+            - name: RABBITMQ_DATA_DIR
+              value: /var/lib/rabbitmq
+            - name: RABBITMQ_MNESIA_BASE
+              value: /var/lib/rabbitmq/mnesia
+            - name: RABBITMQ_LOG_BASE
+              value: /var/log/rabbitmq
```
Comment (arikmaor) [inline charts/tensorleap/templates/rabbitmq-sts.yaml:41]: See if you can move those to the config file please

Comment (roee-shapira-tensorleap) [reply charts/tensorleap/templates/rabbitmq-sts.yaml:41]: Sorry, this isn't supported

**Pull request merged**